src/strategies/delegate-registry-v2/index.ts

```diff
@@ -12,6 +12,7 @@ const DEFAULT_BACKEND_URL = 'https://delegate-registry-backend.vercel.app';
 type Params = {
   backendUrl: string;
   strategies: Strategy[];
+  delegationV1VChainIds?: number[]; // add this to include v1 delegations
```

Comment:
We are not using this anywhere?

Reply:
It is just forwarded to the backend in line 47 and used there.

Reply:
Thanks @manboy-eth missed it